src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -244,6 +256,19 @@ export class OperationDetails {
         this.working(false);
     }
 
+    public async loadExamples(operation: Operation): Promise<void> {
+        const representations = operation.request.meaningfulRepresentations().map(a => {return {...a}});
```

Comment:
why do we need this copy of the object? can't we use directly `operation.request.meaningfulRepresentations()` ?

Reply:
agree thank you.